google/cloud/pubsublite/cloudpubsub/internal/multiplexed_subscriber_client.py

```diff
@@ -40,22 +38,16 @@ class MultiplexedSubscriberClient(SubscriberClientInterface):
     _executor: ThreadPoolExecutor
     _underlying_factory: AsyncSubscriberFactory
 
-    _multiplexer: ClientMultiplexer[SubscriptionPath, StreamingPullFuture]
+    _lock: Lock
+    _live_clients: Set[StreamingPullFuture]
```

Comment:
Previously, the subscriber enforced that there was only one open subscription stream per-subscription per-client, but there's actually no need for this.